kura/org.eclipse.kura.container.provider/src/main/java/org/eclipse/kura/container/provider/ContainerInstance.java

```diff
@@ -49,8 +50,9 @@ public class ContainerInstance implements ConfigurableComponent, ContainerOrches
 
     private ContainerOrchestrationService containerOrchestrationService;
     private Set<ContainerSignatureValidationService> availableContainerSignatureValidationService = new HashSet<>();
-
+    private ConfigurationService configurationService;
     private State state = new Disabled(new ContainerInstanceOptions(Collections.emptyMap()));
+    private ContainerInstanceOptions actualOptions = null;
```

Comment:
Mmmmmmh I think you fell for a classic "false friend" blunder here. You probably wanted to say *current* options, didn't you?

```suggestion
    private ContainerInstanceOptions currentOptions = null;
```

Reply:
I did it **110%**